File: backend/app/plugins/simulators/chat/manifest.py
from app.plugins.base import PluginCategory, PluginManifest

manifest = PluginManifest(
    code="chat",
    name="Chat",
    name_en="Chat",
    description="Communication en temps reel",
    category=PluginCategory.SIMULATOR,
    icon="MessageCircle",
    default_color="teal",
    supported_formats=["text"],
    default_enabled=False,
    coming_soon=False,
    sort_order=5,
    inject_types=[],
    router_module="app.routers.simulated_channels",
    router_prefix="/api/simulated",
)

File: frontend/src/plugins/simulators/chat/manifest.ts
import { lazy } from 'react'
import { MessageCircle } from 'lucide-react'
import type { PluginManifest } from '../../types'

export const manifest: PluginManifest = {
  code: 'chat',
  name: 'plugins.chat.name',
  description: 'plugins.chat.description',
  category: 'simulator',
  icon: MessageCircle,
  defaultColor: 'teal',
  supportedFormats: ['text'],
  defaultEnabled: false,
  comingSoon: false,
  sortOrder: 5,
  injectTypes: [],
  playerRoute: '/chat',
  PlayerPage: lazy(() => import('./PlayerPage')),
  EventReceiver: lazy(() => import('./EventReceiver')),
  PreviewPopup: lazy(() => import('./PreviewPopup')),
  configSchema: null,
}
